index.js

```diff
@@ -3,8 +3,8 @@ import { NativeModules } from 'react-native';
 const { GoogleCast } = NativeModules;
 
 export default {
-  startDefaultScan: function () {
-	GoogleCast.startDefaultScan();
+  startScan: function () {
```

Comment:
we can't have this key repeated.

Instead, use the `maybe` type from flow in the `startScan` that already exists.

`startScan: function (appId: ?string)`

This means that appId could be an string, a null or an undefined value. So we cover both cases